src/components/dashboard/sidebar/EnterpriseOffersSummaryCard.jsx

```diff
@@ -31,7 +31,26 @@ const EnterpriseOffersSummaryCard = ({ className, searchCoursesCta }) => (
     }
     cardClassNames={className}
   >
-    {ENTERPRISE_OFFER_SUMMARY_CARD_SUMMARY}
+    {offer.remainingBalanceForUser
+      ? (
+        <p data-testid="offer-summary-text-detailed">
+          Apply your <b>${offer.remainingBalanceForUser}</b>{' '}
```

Comment:
I'm not sure if this is a thing but I will ask it anyway: do we have a preference for <strong> over <b>? 🤷